src/pick.c

```diff
@@ -48,6 +49,7 @@ struct choice {
 	char	*description;
 	char	*string;
 	size_t	 length;
+	size_t	 printable;
```

Comment:
Could it be renamed to `printable_length` for consistency and add a comment on the definition of a printable character?